package.json

```diff
@@ -38,24 +35,21 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git@github.com:sveltestrap/sveltestrap.git"
+    "url": "https://github.com/sveltestrap/sveltestrap.git"
   },
   "files": [
     "dist",
     "src"
   ],
   "exports": {
     ".": {
-      "types": "./src/index.d.ts",
-      "svelte": "./src/index.js",
-      "require": "./dist/index.js",
-      "module": "./dist/index.ejs",
-      "import": "./dist/index.ejs"
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js"
     }
   },
   "scripts": {
     "start": "storybook dev -p 6006",
-    "build": "vite build",
+    "build": "svelte-package --input ./src && publint --strict",
```

Comment:
```suggestion
    "build": "pnpm lint:package && svelte-package --input ./src",
```

Reply:
Publint needs to run after the build output has been generated, so I've put it as the second entry.